README.md

```diff
@@ -164,16 +135,16 @@ informative video: https://www.loom.com/share/20f9ea27b8da4104acf74943cbc11b7b
  ACKNOWLEDGEMENTS 
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I want to thank <a href="https://www.behance.net/adagio07">Cindy Shin<a> the <a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015"> author of the original design </a>, who provided the behance design of the template
 
-I would like to thank to Alejandro who tough me  how to write good code on css and also, Marco who was always supportive
-
+-Also, I would like to thank to Alejandro who tough me  how to write good code on css. 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
  LICENSE 
```

Comment:
- [OPTIONAL] Good job implementing the changes 🥇 I think it would be better if you consider removing line 143 and similar text from the README. This is because a corresponding heading is already present. This will help you to document your README professionally 😃I know you can do that 💪 See below for more clarity 👇
![image](https://user-images.githubusercontent.com/106119105/225766731-a8351d21-acf3-4b58-8516-c76efe74d3b4.png)

- [OPTIONAL] Please consider adding the **_Show you support_** and **_Contributing_** sections in the README. I noticed that these are present in the table of the content but a corresponding descriptive section is missing ⚒ Let's fix that as well. What do you think? 😃